### ISSUE TITLE
Enable support for iTunes encrypted Backup (iPhone) + Improve UX

### ISSUE THREAD

**Issue by koalalorenzo** [bitpay/copay#6716]
TL;DR: I have just lost some coins. Not enough to cry but enough to remind other people to do proper backups, and to not rely on one single backup format (try always to have the memphrase, not the json!).

## What happened? 
I always do backup, and most of the time I try to have the memphrase... but this specific wallet that I made at the beginning of the year was exported and encrypted using the json format. 
## What happened then?
In June I switched from iOS 10 to iOS 11 Beta. I made an encrypted backup of iOS 10 with iTunes. When then I restored to iOS 10 after the beta, I created a new wallet and then I restored the json files. ( Somehow I had no clue the json file was just a pub key ). 

I bought some coins and got the address from copay. I thought I had even selected the brand new wallet that I created after going back to iOS 10, but apparently I got the old wallet generated in January. Without private key.

Money arrived. I tried to send them and I got a "Payment proposal". Panic, Tried to restore somewhere again with the same json... and still got only a pub key.

So I tried to restore the backup from iTunes, and it apparently restored all the content in most of the apps... but not Copay: if this is a security reason, it sucks because that was my last shore.

So my features request includes:

- Restore/improve the UX, the previous Copay user experience was much better! the new Bitpay-based wallet experience is really confusing when managing multiple wallets!
- Improve the Icons and their sizes: It is not clear when "receiving" on a wallet if you do or do not have the private keys...
- PLZ Get rid of the json export wallets: they are just confusing and apparently I was not aware that I made a backup of the pub key!!!
- Encrypt your wallet and allows at least iTunes Encrypted Backups. 

I am ready to help again and translate the app, but please consider this as some important flaws in the UX! Hopefully somebody will hear me, or help me somehow...

**Comment by mac12345678** [comment]
was this feature ever developed trying to restore wallet from itunes at the moment?
